internal/pkg/auth/user_login.go

```diff
@@ -135,8 +135,9 @@ func AuthorizeUser(p *print.Printer, isReauthentication bool) error {
 
 		// Get the authorization code
 		code := r.URL.Query().Get("code")
+		errDescription := r.URL.Query().Get("error_description")
 		if code == "" {
-			errServer = fmt.Errorf("could not find 'code' URL parameter")
+			errServer = fmt.Errorf("could not find 'code' URL parameter.\nError: %s", errDescription)
```

Comment:
```suggestion
			errServer = fmt.Errorf("could not find 'code' URL parameter: %s", errDescription)
```

And probably also check and only concatenate `errDescription` if it is not empty

Reply:
Added the check